client/public/index.html

```diff
@@ -28,6 +28,7 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+    <script src="https://cdn.jsdelivr.net/gh/alpinejs/alpine@v2.x.x/dist/alpine.min.js" defer></script>
```

Comment:
What is alpinejs?

Reply:
its just what i need to make the basket work, its not a dependancy its just a script tag on the index.html